### DISCUSSION TITLE
No more h264?

### DISCUSSION THREAD

**Discussion by kursad-k** [ggarra13/mrv2#209] [Q&A] [answered]
Hi

I wanted to convert a sequence into a movie to .mp4, but I see that other format types have been removed. Do you have plans to bring back h264?

This is what I see on windows 11

![image](https://github.com/ggarra13/mrv2/assets/67970138/812b9bcc-67d1-4b98-9d7d-be8b20dfd198)

**Comment by ggarra13** [comment, marked as answer]
H264 was there but it was not functional, as it has a conflicting license.  You *can* get it back if you compile in GPL mode from source, with:

`
runme.sh -gpl 
`

You can get a .mp4 file if you use some other codec like VP9 or AV1.

**Reply by kursad-k** [reply]
Won't these codecs be harder to support in the older pieces of hardware? I would like the output to be as widely supported as possible.

**Reply by ggarra13** [reply]
Hardware should not be a problem.  VP9 and AV1 are both really efficient with decoding resources.  

What might get you is software compatibility, specially on the web.   While all web browsers support them without issues, some web pages might not support them.  Youtube, for example, supports VP9 but I think does not yet support AV1.

**Reply by kursad-k** [reply]
I think IOS is going to have hard time with these, at least I had playback issues with vp9 on IOS last year.

**Reply by ggarra13** [reply]
This is what google had to say:

Android has supported VP9 since version 4.4 KitKat, while iOS/iPadOS added support for VP9 in iOS/iPadOS 14.

If that's not good enough, you can save with "None", which should really be named mpeg4.  It is the oldest codec available and everything should support  it.

**Reply by kursad-k** [reply]
Thanks for your help, I will investigate these options.
